docs/astro/src/content/docs/reference/elements/image.mdx

```diff
@@ -251,3 +251,29 @@ Use the <Link type="ImageType" label="`@image-url` macro" /> to specify the imag
 
 Properties in source image coordinates that define the region of the source image that is rendered.
 By default the entire source image is visible:
+
+## Accessibility
```

Comment:
I'd mention somewhere in this paragraph that Image have a default accessible-role